drivers/serial/uart_nrfx_uarte.c

```diff
@@ -275,7 +275,11 @@ static int uarte_nrfx_configure(struct device *dev,
 	nrf_uarte_parity_t parity;
 	nrf_uarte_hwfc_t hwfc;
 
-	if (cfg->stop_bits != UART_CFG_STOP_BITS_1) {
+	if (!(cfg->stop_bits == UART_CFG_STOP_BITS_1
+#ifdef UARTE_CONFIG_STOP_Two
+	      || cfg->stop_bits == UART_CFG_STOP_BITS_2
+#endif
+	      )) {
```

Comment:
```
switch (cfg->stop_bits) {
...
default:
	return -ENOTSUP;
}
```
will be a bit clearer, and it will be consistent with what we've got below for other settings. You can use, for instance, `bool two_stop_bits` to store configuration of stop bits for now and replace it later with what will be required by the updated nrfx HAL (it will result in less changes after the nrfx update).